ambry-rest/src/test/java/com.github.ambry.rest/NettyMessageProcessorTest.java

```diff
@@ -183,7 +183,6 @@ public void requestHandleWithBadInputTest()
     channel.writeInbound(new DefaultLastHttpContent(Unpooled.wrappedBuffer(content.getBytes())));
     HttpResponse response = (HttpResponse) channel.readOutbound();
     assertEquals("Unexpected response status", HttpResponseStatus.BAD_REQUEST, response.getStatus());
-    assertFalse("Channel is still active", channel.isActive());
```

Comment:
I don't think that is necessary. Like I said, the check does not make sense here in `NettyMessageProcessorTest` (which is the unit test for `NettyMessageProcessor`) because if the check fails, the fix is in `NettyResponseChannel` and not in `NettyMessageProcessor`.